core/templates/pages/exploration-editor-page/translation-tab/services/translation-status.service.ts

```diff
@@ -135,7 +136,15 @@ angular.module('oppia').factory('TranslationStatusService', [
           if (!interactionId ||
             INTERACTION_SPECS[interactionId].is_linear ||
             INTERACTION_SPECS[interactionId].is_terminal) {
-            allContentId = ['content'];
+            var contentIdToRemove = _getContentIdListRelatedToComponent(
+              COMPONENT_NAME_HINT);
+            // Excluding default_outcome content status as default outcome's
+            // content is left empty so the translation or voiceover is not
+            // required.
+            contentIdToRemove.push('default_outcome');
+            allContentId = allContentId.filter(function(contentId) {
```

Comment:
should this be `allContentIds` because it is a list?

Reply:
`allContentIds` was an existing variable. (But I agree that it should be plural, I've changed it.)